src/main/java/org/jenkinsci/plugins/workflow/support/steps/ExecutorStepExecution.java

```diff
@@ -1189,8 +1153,6 @@ public String getAbsoluteUrl() {
                 return "PlaceholderExecutable:" + PlaceholderTask.this;
             }
 
-            private static final long serialVersionUID = 1L;
```

Comment:
Unclear why this was here, when the `PlaceholderExecutable` is not `Serializable`.